sqlite.go

```diff
@@ -993,16 +1094,45 @@ func (t *tx) exec(stmt string) (err error) {
 type conn struct {
 	*sqlite
 	*virtual.Thread
-	pdb uintptr
+	loc           *time.Location
+	compatibility string
+	pdb           uintptr
 }
 
 func (c *conn) String() string {
 	return fmt.Sprintf("&%T@%p{sqlite: %p, Thread: %p, pdb: %#x}", *c, c, c.sqlite, c.Thread, c.pdb)
 }
 
-func newConn(s *sqlite, name string) (_ *conn, err error) {
+func newConn(s *sqlite, dsn string) (_ *conn, err error) {
 	c := &conn{sqlite: s}
 
+	pos := strings.IndexRune(dsn, '?')
+	fmt.Println("DSN: ", dsn)
```

Comment:
For debugging I use the dbg function (it's defined in all_test.go). It works only while testing but the package does not build otherwise.